homeassistant/components/ads/__init__.py

```diff
@@ -46,7 +52,8 @@
 
 SCHEMA_SERVICE_WRITE_DATA_BY_NAME = vol.Schema({
     vol.Required(CONF_ADS_TYPE):
-        vol.In([ADSTYPE_INT, ADSTYPE_UINT, ADSTYPE_BYTE, ADSTYPE_BOOL]),
+        vol.In([ADSTYPE_INT, ADSTYPE_UINT, ADSTYPE_BYTE, ADSTYPE_BOOL, /
```

Comment:
Break the line after the bracket instead. Make sure to end each bracket or parenthesis with the corresponding bracket or parenthesis.